<?xml version="1.0"?>
<launch>
    <arg name="file"  default="$(find simulation)/scripts/markers.csv" />
	<node pkg="simulation" type="objective6.py" name="objective6" output="screen">
        <param name="file" value="$(arg file)" />
	</node>
</launch>
